examples/speech_recognition/model/README.md

```diff
@@ -0,0 +1,20 @@
+# Inter OpenVINO
```

Comment:
Inter -> Intel?

Reply:
> Inter -> Intel?

Yes, I will modify this and resubmit.

Reply:
Probabaly we don't need this title since this is not about "Intel OpenVINO".